First commit

heintayzar-hm opened the pull request:
First PR from trials.

Thank you for viewing my PR.

heintayzar-hm committed:
First commit

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
```
I think it would be better to use semantic tags <header>

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
```
I think it would be better to use semantic tags `<header class="basis-1/4">` because it is for the header section and it is good for semantic reasons and make you see more clearly.😃😃😃
[For reference:](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Santiago220991 replied:
- I think you did a great job using the inline comments. :clap: :clap: However please remember to always include an explanation, a "why", for each requested change. Some things could be obvious for you but not for someone else :smiley: :smiley:
- [OPTIONAL] I think it's a good idea to use some emojis to make your review more human-friendly. :space_invader: :space_invader:

karlavdelgadof replied:
- [ ] As mentioned by the previous reviewer, providing a reason for the change will help the author of the PR to quickly understand and identify :bulb:  the change/bug or improvement to be resolved in their code. 

In the case of this comment, please make sure to elaborate after you use "because"  :heavy_check_mark: 
> I think it would be better to use semantic tags `<header>` because it is for the header section. blushblushblushblushblush For reference: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

karlavdelgadof replied:
> I think it would be better to use semantic tags `<header>` because it is for the header section. blushblushblushblushblush For reference: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

**[OPTIONAL]:** You could easily add this link to more information in a more visually organized way by making use of the GitHub Markdown. Here's a link with some [useful tips to make your reviews and PRs look aesthetically more professional :necktie: :100: ](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

heintayzar-hm replied:
@karlavdelgadof  Is that okay? Do I need to add something?

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+          id="navbar"
+          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
+        >
+          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
```
I think it would be better to use semantic tags <hx> in order x. Please change the following ones too. We have to use header <hx> tags as this is good for semantic purposes and make your code more readable.🥂🥂🥂🥂
[For reference:](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Santiago220991 replied:
- Same here kindly remember to always include an explanation, a "why". [OPTIONAL] Also please try to include some emojis It is always a good idea. :eyes:

karlavdelgadof replied:
Great job adding some emojis :raised_hands: ! But also please be careful of their usability as some can be easily misinterpreted :eyes: :warning: !

karlavdelgadof replied:
> I think it would be better to use semantic tags `<h1>`. Cause it is a header text for the entire page.shrugshrugshrugshrug For reference:https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

- [ ] Great job catching this improvement, however, as mentioned before, please elaborate the reason behind this change :+1:

heintayzar-hm replied:
@karlavdelgadof Is that okay? Do I need to add something?

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
+            Our Technical Page
+          </header>
+          <div
+            class="flex flex-col justify-evenly space-x-2 overflow-y-auto overflow-x-hidden"
+          >
+            <a href="#our_project" class="nav-link uppercase">our project</a>
+            <a href="#html" class="nav-link uppercase">html</a>
+            <a href="#css" class="nav-link uppercase">css</a>
+            <a href="#js" class="nav-link uppercase">js</a>
+            <a href="#php" class="nav-link uppercase">php</a>
+          </div>
```
I think it would be better to use semantic tags `<nav>`.
`<nav>` is a navigational section as this is recommended for semantic reasons and to make your code clearer. (it is a collection of links for the websites).
[📖📖📖📖 For reference:](https://www.w3schools.com/tags/tag_nav.asp)

karlavdelgadof replied:
> I think it would be better to use semantic tags `<nav>`. `<nav>` is a navigational section. (literally, it is a collection of links for the websites). bookbookbookbook For reference: https://www.w3schools.com/tags/tag_nav.asp

- [ ] Same in here, for any comment/suggestion you provide, please make sure to include an explanation :memo: :exclamation:, and be careful when using expressions like "literally" as this might sound frustrating :thinking:

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+
+      <div class="basis-3/4">
+        <section class="main-section" id="our_project">
+          <header class="uppercase font-bold">our project</header>
```
I think it would be better to use semantic tags `<hx>` in order x. Please change the following ones too. We have to use header text for `<hx>` tags as this is good for semantic purposes

karlavdelgadof replied:
Great job using inline comments and pointing out the rest of the changes instead of repeating the same comment! :clap: :100:

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+      <div class="basis-3/4">
+        <section class="main-section" id="our_project">
+          <header class="uppercase font-bold">our project</header>
+          <p>This is the project we have done</p>
+          <p>We have done best</p>
```
I think it would be better to use semantic tags `<span>`.
📖📖📖📖 We use `<p>` tag for the paragraphs and for the short ones we should use `<span>` tag. This is also good for semantic reasons.

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+          <p>We have done best</p>
+        </section>
+        <section class="main-section" id="html">
+          <header class="uppercase font-bold">html</header>
```
I think it would be better to use semantic tags <hx> x in order

heintayzar-hm commented on index.html:
```diff
@@ -0,0 +1,131 @@
+          <p>That is javascript</p>
+          <code> x=5; y=6; z=x+y; </code>
+          <code></code>
+          <code></code> <code></code> <code></code>
```
To be more readable, you should add something on the line 84 and 85 (you cannot just delete as it is needed for the project), in this way you can make your code more cleaner and contains all the necessary information

[📖📖📖📖Reference:](https://www.w3schools.com/html/html_computercode_elements.asp)

karlavdelgadof replied:
- [ ] In here, I would suggest providing a more descriptive suggestion like:

"For better code readability, I would suggest either removing these unused elements or filling them up with some content, in this way you can make sure your project looks cleaner and contains all the necessary information :heavy_check_mark: "

This comment provides an explanation and also a pro consequence of implementing the change :100:

heintayzar-hm replied:
@karlavdelgadof  Is that okay? Do I need to add something?

heintayzar-hm reviewed (commented):
Hi @heintayzar-hm , Good job.

<img src="https://media1.giphy.com/media/5QW76Ww9bquHdg1fTv/giphy.gif"/>

Status: Required Changes. ♻️

Good point:💖💖💖
Pass All tests.

Needed to change:🙉🙉🙉
Please use linters.
And add readme file.

Santiago220991 reviewed (changes requested):
Hi @heintayzar-hm , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


<img src="https://media4.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
:heavy_check_mark: Polite comments.
:heavy_check_mark: Inline comments.
:heavy_check_mark: Using GitHub review features.

## Required Changes ♻️

- Kindly edit all your requested changes adding an explanation of why you requested those changes. Please remember that some things could be obvious for you but not for someone else. :+1: :smiley: :smiley: 

_Check also the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly add emojis to your comments. It makes your review more human-friendly. 
- It is also a good idea to include a GIF image in your summary as I did in this summary :smiley:  To do this you could install an extension for Chrome called "GIFs for GitHub". :nerd_face: :nerd_face:  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

karlavdelgadof reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle: 

Hi there @heintayzar-hm :technologist: :wave: ! , 

Great job introducing the changes requested by the previous reviewer!👏 
You've done well implementing some of the requested changes, but there are still some issues that still need to be addressed for your project to shine :sparkles:.

![](https://media.giphy.com/media/xUOxfh6ZM75efM3Bqo/giphy.gif)

## :sparkles: Highlights :sparkles: 

- Great work implementing emojis to make your review human-friendly :100: 
- The comments provide links to useful info :heavy_check_mark: 

Congrats on your work so far! :clap:  


 ## Required Changes ♻️

 There are some issues that you still need to work on, but you are nearly there! 
_For more information, check the comments under the review :arrow_down:_


## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@karlavdelgadof**  in your question so I can receive the notification.
  
  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
  ------
  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

heintayzar-hm commented:
Can you take a look at my review? Because if I failed this time, I have to
request an extra code review.


On Mon, Sep 26, 2022, 20:22 Karla Delgado ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRED CHANGES ♻️
>
> Hi there @heintayzar-hm <https://github.com/heintayzar-hm> 🧑‍💻 👋 ! ,
>
> Great job introducing the changes requested by the previous reviewer!👏
> You've done well implementing some of the requested changes, but there are
> still some issues that still need to be addressed for your project to shine
> ✨.
> ✨ Highlights ✨
>
>    - Great work implementing emojis to make your review human-friendly 💯
>    - The comments provide links to useful info ✔️
>
> Congrats on your work so far! 👏
> Required Changes ♻️
>
> There are some issues that you still need to work on, but you are nearly
> there!
> *For more information, check the comments under the review ⬇️*
> Optional suggestions ☑️
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you take them into
> account as they can improve the quality of your code 💯*
>
> Happy coding and keep it up! 🙌
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me ***@***.***
> <https://github.com/karlavdelgadof>* in your question so I can receive
> the notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980042073>
> :
>
> > +  <head>
>
> +    <meta charset="UTF-8" />
>
> +    <title>Fork Me! FCC: Test Suite Template</title>
>
> +    <link rel="stylesheet" href="css/style.css" />
>
> +    <script src="https://cdn.tailwindcss.com"></script>
>
> +    <script>
>
> +      tailwind.config = {
>
> +        theme: { extend: { colors: { clifford: "#da373d" } } },
>
> +      };
>
> +    </script>
>
> +  </head>
>
> +  <body>
>
> +    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
>
> +    <main id="main-doc" class="flex flex-row">
>
> +
>
> +      <div class="basis-1/4">
>
>
>
>    - As mentioned by the previous reviewer, providing a reason for the
>    change will help the author of the PR to quickly understand and identify
>    💡 the change/bug or improvement to be resolved in their code.
>
> In the case of this comment, please make sure to elaborate after you use
> "because" ✔️
>
> I think it would be better to use semantic tags <header> because it is
> for the header section. blushblushblushblushblush For reference:
> https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md
>
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980044429>
> :
>
> > +  <head>
>
> +    <meta charset="UTF-8" />
>
> +    <title>Fork Me! FCC: Test Suite Template</title>
>
> +    <link rel="stylesheet" href="css/style.css" />
>
> +    <script src="https://cdn.tailwindcss.com"></script>
>
> +    <script>
>
> +      tailwind.config = {
>
> +        theme: { extend: { colors: { clifford: "#da373d" } } },
>
> +      };
>
> +    </script>
>
> +  </head>
>
> +  <body>
>
> +    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
>
> +    <main id="main-doc" class="flex flex-row">
>
> +
>
> +      <div class="basis-1/4">
>
>
> I think it would be better to use semantic tags <header> because it is
> for the header section. blushblushblushblushblush For reference:
> https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md
>
> *[OPTIONAL]:* You could easily add this link to more information in a
> more visually organized way by making use of the GitHub Markdown. Here's a
> link with some useful tips to make your reviews and PRs look
> aesthetically more professional 👔 💯
> <https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax>
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980047862>
> :
>
> > +    <script>
>
> +      tailwind.config = {
>
> +        theme: { extend: { colors: { clifford: "#da373d" } } },
>
> +      };
>
> +    </script>
>
> +  </head>
>
> +  <body>
>
> +    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
>
> +    <main id="main-doc" class="flex flex-row">
>
> +
>
> +      <div class="basis-1/4">
>
> +        <nav
>
> +          id="navbar"
>
> +          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
>
> +        >
>
> +          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
>
>
> Great job adding some emojis 🙌 ! But also please be careful of their
> usability as some can be easily misinterpreted 👀 ⚠️ !
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980049414>
> :
>
> > +    <script>
>
> +      tailwind.config = {
>
> +        theme: { extend: { colors: { clifford: "#da373d" } } },
>
> +      };
>
> +    </script>
>
> +  </head>
>
> +  <body>
>
> +    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
>
> +    <main id="main-doc" class="flex flex-row">
>
> +
>
> +      <div class="basis-1/4">
>
> +        <nav
>
> +          id="navbar"
>
> +          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
>
> +        >
>
> +          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
>
>
> I think it would be better to use semantic tags <h1>. Cause it is a
> header text for the entire page.shrugshrugshrugshrug For reference:
> https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md
>
>
>    - Great job catching this improvement, however, as mentioned before,
>    please elaborate the reason behind this change 👍
>
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980055617>
> :
>
> > +          <div
>
> +            class="flex flex-col justify-evenly space-x-2 overflow-y-auto overflow-x-hidden"
>
> +          >
>
> +            <a href="#our_project" class="nav-link uppercase">our project</a>
>
> +            <a href="#html" class="nav-link uppercase">html</a>
>
> +            <a href="#css" class="nav-link uppercase">css</a>
>
> +            <a href="#js" class="nav-link uppercase">js</a>
>
> +            <a href="#php" class="nav-link uppercase">php</a>
>
> +          </div>
>
>
> I think it would be better to use semantic tags <nav>. <nav> is a
> navigational section. (literally, it is a collection of links for the
> websites). bookbookbookbook For reference:
> https://www.w3schools.com/tags/tag_nav.asp
>
>
>    - Same in here, for any comment/suggestion you provide, please make
>    sure to include an explanation 📝 ❗, and be careful when using
>    expressions like "literally" as this might sound frustrating 🤔
>
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980056952>
> :
>
> > +          </header>
>
> +          <div
>
> +            class="flex flex-col justify-evenly space-x-2 overflow-y-auto overflow-x-hidden"
>
> +          >
>
> +            <a href="#our_project" class="nav-link uppercase">our project</a>
>
> +            <a href="#html" class="nav-link uppercase">html</a>
>
> +            <a href="#css" class="nav-link uppercase">css</a>
>
> +            <a href="#js" class="nav-link uppercase">js</a>
>
> +            <a href="#php" class="nav-link uppercase">php</a>
>
> +          </div>
>
> +        </nav>
>
> +      </div>
>
> +
>
> +      <div class="basis-3/4">
>
> +        <section class="main-section" id="our_project">
>
> +          <header class="uppercase font-bold">our project</header>
>
>
> Great job using inline comments and pointing out the rest of the changes
> instead of repeating the same comment! 👏 💯
> ------------------------------
>
> In index.html
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#discussion_r980062101>
> :
>
> > +            <li>5</li>
>
> +          </ul>
>
> +        </section>
>
> +        <section class="main-section" id="js">
>
> +          <header class="uppercase font-bold">js</header>
>
> +
>
> +          <p>
>
> +            JavaScript Can Change HTML Content One of many JavaScript HTML
>
> +            methods is getElementById(). The example below "finds" an HTML
>
> +            element (with id="demo"), and changes the element content
>
> +            (innerHTML) to "Hello JavaScript":
>
> +          </p>
>
> +          <p>That is javascript</p>
>
> +          <code> x=5; y=6; z=x+y; </code>
>
> +          <code></code>
>
> +          <code></code> <code></code> <code></code>
>
>
>
>    - In here, I would suggest providing a more descriptive suggestion
>    like:
>
> "For better code readability, I would suggest either removing these unused
> elements or filling them up with some content, in this way you can make
> sure your project looks cleaner and contains all the necessary information
> ✔️ "
>
> This comment provides an explanation and also a pro consequence of
> implementing the change 💯
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/heintayzar-hm/review-past-projects/pull/1#pullrequestreview-1120239338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZCNHXNQYRTY6G3NYQMTVGDWAGTDRANCNFSM6AAAAAAQVW6QSM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ichala reviewed (approved):
Hi @manq2010 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

- Changes implemented ✔️ 

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
